src/redux/actionTypes.ts

```diff
@@ -180,6 +183,7 @@ export enum ActionTypes {
   MULTISIG_FINALIZE = 'MULTISIG_FINALIZE',
   MULTISIG_FINALIZE_ERROR = 'MULTISIG_FINALIZE_ERROR',
   MULTISIG_FINALIZE_SUCCESS = 'MULTISIG_FINALIZE_SUCCESS',
+  MULTISIG_MANAGE_VERIFIED_MEMBERS = 'MULTISIG_MANAGE_VERIFIED_MEMBERS',
```

Comment:
is this obsolete by chance? since we're using the same motion for multisig and voting reputation